src/bindings/pygaia/scripts/classification/classification_project_template.yaml

```diff
@@ -88,16 +100,25 @@ preprocessing:
         - transfo: gaussianize
           params: { descriptorNames: 'lowlevel.*' }
 
+    mfcc:
+        # an MFCC only baseline
+        - transfo: remove
+          params: { descriptorNames: *unusedDescs }
+        - transfo: enumerate
+          params: { descriptorNames: *stringDescs }
+        - transfo: select
+          params: { descriptorNames: 'lowlevel.mfcc*' }
 
 # List of classifiers to be trained
 classifiers:
     svm:
         # first svm test combinations
-        - preprocessing: [ 'basic', 'lowlevel', 'nobands', 'normalized', 'gaussianized' ] # 'nosfx'
+        - preprocessing: [ 'basic', 'lowlevel', 'nobands', 'normalized', 'gaussianized', 'mfcc' ] # 'nosfx'
           type: [ 'C-SVC' ]
           kernel: [ 'poly', 'RBF' ]
           C:     [ -5, -3, -1, 1, 3, 5, 7, 9, 11 ] # will actually be 2**x
           gamma: [ 3, 1, -1, -3, -5, -7, -9, -11 ] # will actually be 2**x
+          balanceClasses: [False, True] # whether to weight the classes according to the number of elements or not
```

Comment:
Can you clarify this comment? Does this configuration mean that it will run twice the number of parameters as before, one with this True and one with it False?

Reply:
Agreed, the comment should be improvement. This is a parameter of SVM training. According to this code the training is run twice. @pabloEntropia